qucs/qucs/imagewriter.cpp

```diff
@@ -156,6 +156,16 @@ int ImageWriter::print(QWidget *doc)
   if (dlg->exec()) {
     QString filename = dlg->FileToSave();
     lastExportFilename = filename;
```

Comment:
nit-picking/question: should the `lastExportFilename` be updated only if actually used successfully, i.e. move the assignment after the `if (r == QMessageBox::No)` check?